app/language/language-android/src/main/kotlin/com/hedvig/android/language/AndroidLanguageService.kt

```diff
@@ -39,9 +39,9 @@ internal class AndroidLanguageService() : LanguageService {
   }
 
   private fun getLocaleFromAppCompat(): Locale? {
-    val localeList = AppCompatDelegate.getApplicationLocales()
+    val localeList = AppCompatDelegate.getApplicationLocales() // here: weirdly depends on cache
     return if (localeList.isEmpty) {
-      null
+      Locale.getDefault()
```

Comment:
This would also make the distinction between `getLocale` and `getSelectedLocale` be more weird. Since before, "selected" variant returned null if we did not know what it was. Now it would return the system default while the other variant would default to "en-SE" but it'd be quite unclear regarding why the behavior is the way it is and why would one use one over the other